Tella/Scenes/Settings/Views/Common/ResizableImage.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  ResizableImage.swift
+//  Tella
+//
+//  Created by RIMA on 14.11.24.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import SwiftUI
+
+struct ResizableImage: View {
```

Comment:
Could it be a shared view used across all modules, not just for settings?

Reply:
Yes, it should be shared. ILL just move the position